scala-ts-mode.el

```diff
@@ -632,9 +645,7 @@ Return nil if there is no name or if NODE is not a defun node."
                                                  (import extra)
                                                  (variable function operator literal interpolation)))
 
-
-    (setq-local
-     treesit-simple-indent-rules scala-ts--indent-rules)
+    (add-hook 'hack-local-variables-hook #'scala-ts--setup-indentation nil t)
```

Comment:
I'm not sure if this is called everytime, even if the project doesn't have any local variables? Can you please verify